README.md

```diff
@@ -243,13 +243,21 @@ As long as a key hasn't been directly defined, you may still write to it and
 to names within it.
 
 ```
+# This makes the key "fruit" into a table.
```

Comment:
Maybe, to be more precise:

> This makes the keys fruit and fruit.apple into tables.

Or even:

> This makes the keys fruit and fruit.apple into tables (one nested inside the other).

Reply:
Thanks, but for this one example, I want focus to be on the key `fruit`. True that `fruit.apple` is also made to be a table. But I don't want to discuss how to define tables as much as I want to show how keys' value types remain the same.